src/.clang-format

```diff
@@ -12,7 +12,11 @@ AlwaysBreakBeforeMultilineStrings: false
 AlwaysBreakTemplateDeclarations: true
 BinPackParameters: false
 BreakBeforeBinaryOperators: false
-BreakBeforeBraces: Linux
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass: true
+  AfterFunction: true
+  AfterStruct: true
```

Comment:
structs had their braces on the same line previously?

Reply:
> structs had their braces on the same line previously?

Good catch, this was a mistake.